grafana/risingwave-dev-dashboard.dashboard.py

```diff
@@ -622,6 +622,11 @@ def section_object_storage(outer_panels):
 def section_streaming(panels):
     mv_filter = "executor_identity=~\".*MaterializeExecutor.*\""
     sink_filter = "executor_identity=~\".*SinkExecutor.*\""
+    executor_identity = '.*MaterializeExecutor.*'
+    job = '$job'
+    instance = '$node'
+    mv_throughput_query = f'sum(rate(stream_executor_row_count{{executor_identity=~"{executor_identity}", job=~"{job}", instance=~"{instance}"}}[$__rate_interval]) * on(actor_id) group_left(id, mv_name) materialized_info_with_actor_id) by (id, mv_name)'
```

Comment:
we need to call metric fn on stream_executor_row_count and we don't need to explicitly add the job and instance filters.